File: backend_project/src/main/java/com/ssafy/model/entity/Schedule.java
package com.ssafy.model.entity;

import java.time.LocalDateTime;

import com.ssafy.model.entity.Room;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Schedule {
    private Long scheduleId;
    private Long roomId;
    private LocalDateTime startTime;
    private LocalDateTime endTime;
    private String content;
    // 연관
    private Room room;
}

File: TMBG/app/src/main/java/com/ssafy/tmbg/data/schedule/dao/Schedule.kt
package com.ssafy.tmbg.data.schedule.dao

import java.util.*

/**
 * 일정 정보를 담는 데이터 클래스
 * 서버와의 통신 및 앱 내부에서 사용됩니다.
 *
 * @property scheduleId 일정의 고유 ID
 * @property roomId 일정이 속한 방의 ID
 * @property startTime 일정 시작 시간
 * @property endTime 일정 종료 시간
 * @property content 일정 내용
 */
data class Schedule(
    val scheduleId: Int,
    val roomId: Int,
    val startTime: String,
    val endTime: String,
    val content: String
)

